native/common/jp_bytetype.cpp

```diff
@@ -16,9 +16,13 @@
 *****************************************************************************/   
 #include <jp_primitive_common.h>
 
+#define FIELD(X) X.b
```

Comment:
why not use templates in the first place? Adding macros does not make the code more failsafe nor readable.

Reply:
I can make it a template.  I was just looking for a quick way to make the differences between these different primitives when using meld stand out.  My next set of changes gets rid of all the RETHROW_CATCH statements so removing the macros is my overall goal.